projects/storefrontlib/context/routing-context.service.ts

```diff
@@ -47,7 +47,7 @@ export class RoutingContextService {
    */
   get<T>(contextToken: ContextToken): Observable<T | undefined> {
     return this.contextTokenMapping$.pipe(
-      map((contextMapping) => {
+      switchMap((contextMapping) => {
```

Comment:
was this a bug in the original implementation?

Reply:
no, `UnifiedInjector`'s get return `Observable, but Injector.get not.

Reply:
Thank! I didn't know that.